concepts/src/controllers/concept.ts

```diff
@@ -17,21 +18,26 @@ const conceptController = (
 
   return asyncHandler(async (req, res) => {
     const id = req.params.id;
-    const getResponse = await elasticClient.get<Displayable<Concept>>({
-      index,
-      id,
-      _source: ["display"],
-    });
-
-    if (!getResponse.found || !getResponse._source) {
-      throw new HttpError({
-        status: 404,
-        label: "Not Found",
-        description: `Concept not found for identifier ${id}`,
+    try {
+      const getResponse = await elasticClient.get<Displayable<Concept>>({
+        index,
+        id,
+        _source: ["display"],
       });
-    }
 
-    res.status(200).json(getResponse._source.display);
+      res.status(200).json(getResponse._source!.display);
```

Comment:
What happens if `_source` is missing? In the old code, that would send us down the 404 path – what happens in the new code?

Reply:
It isn't possible for it to be missing if the `get` hasn't thrown, but unfortunately this isn't enforced by the types Elastic provide - it would 500, which I am fine with because it would be very much unexpected.